components/Common/SidebarItem/index.tsx

```diff
@@ -0,0 +1,32 @@
+import classNames from 'classnames';
+import type { FC } from 'react';
+
+import LocalizedLink from '@/components/LocalizedLink';
+import type { SidebarItemType } from '@/types/sidebar';
+
+import styles from './index.module.css';
+
+type SideBarItemProps = SidebarItemType & {
+  isActive?: boolean;
+  handleSideBarItemClick: (title: string) => void;
```

Comment:
Since it is a prop located under `SidebarItem`, there is no need to repeat the component name in the props. IMO for simpler use;

```suggestion
  onClick: (title: string) => void;
```